tools/serverpod_cli/lib/src/create/database_setup.dart

```diff
@@ -6,9 +6,18 @@ import 'package:serverpod_cli/src/logger/logger.dart';
 import 'package:serverpod_cli/src/util/command_line_tools.dart';
 
 class DatabaseSetup {
-  static Future<bool> createDefaultMigration(Directory dir, String name) async {
+  static Future<bool> createDefaultMigration(
+    Directory dir,
+    String name,
+  ) async {
     log.debug('Creating default migration.');
```

Comment:
Maybe we should rephrase this as `Creating initial migration`? since we will actually create one but we won't have the tag.